osquery/tables/system/tests/CMakeLists.txt

```diff
@@ -34,6 +34,10 @@ function(osqueryTablesSystemTestsMain)
     generateOsqueryTablesSystemTestsStartupitemstestsTest()
     generateOsqueryTablesSystemTestsSignaturetestsTest()
   endif()
+
+  if(DEFINED PLATFORM_WINDOWS)
+    generateOsqueryTablesSystemTestsCertificatestestsTest()
```

Comment:
This name collides with an already existing function in the macOS case.
You might want to rename both and add the platform, like `generateOsqueryTablesSystemTestsWindowsCertificatestestsTest` and `generateOsqueryTablesSystemTestsmacOSCertificatestestsTest`